### ISSUE TITLE
Data-mdb-* don't work!! 

### ISSUE THREAD

**Issue by AnupamKhosla** [mdbootstrap/material-design-for-bootstrap#301]
I jus cloned your repo https://github.com/mdbootstrap/material-design-for-bootstrap.git and put a navbar with a dropdown and noticed you need to use `data-toggle="dropdown"` instead of `data-mdb-toggle="dropdown"`.  

Is it a bug?

**Comment by quophiBonney** [comment]
I don't think so, what I think is it came with the updated version there the old one which data-toggle="dropdown" was deprecated. I encountered similar error in bootstrap when I started.
